portal/server/api.js

```diff
@@ -23,23 +23,21 @@ const login = async (username, password) => {
   return { token, loginStatus, user };
 };
 
-const sendSignInLink = async (token) =>
-  axios({
-    method: 'post',
-    url: `${PORTAL_API_URL}/v1/users/me/sign-in-link`,
-    headers: {
-      'Content-Type': 'application/json',
-      Authorization: token,
-    },
-  });
-
-const loginWithSignInLink = async ({ userId, signInToken }) => {
+const sendSignInLink = async (token) => axios({
```

Comment:
* Missing function documentation for both the functions.
* Missing CR after first function definition.